javascript/ql/src/semmle/javascript/Functions.qll

```diff
@@ -2,7 +2,22 @@
 
 import javascript
 
-/** A function as defined either by a function declaration or a function expression. */
+/**
+ * A function as defined either by a function declaration or a function expression.
+ *
+ * Examples:
+ *
+ * ```
+ * function greet() {      // function declaration
+ *   console.log("Hi");
+ * }
+ *
+ * var greet =
+ *   function() {          // function expression
+ *     console.log("Hi");
+ *   };
+ * ```
+ */
```

Comment:
I think it's worth including more cases here (can be copied from the subclasses below):
- Arrow function
- Object literal method
- Object literal accessor
- Class method

Reply:
I've amended the commit to include these examples.